modules/alloy/src/main/ts/ephox/alloy/api/ui/SplitFloatingToolbar.ts

```diff
@@ -86,6 +86,8 @@ const factory: CompositeSketchFactory<SplitFloatingToolbarDetail, SplitFloatingT
           FloatingToolbarButton.toggle(floatingToolbarButton);
         });
       },
+      isOn: (toolbar: AlloyComponent) =>
+        memFloatingToolbarButton.getOpt(toolbar).map((floatingToolbarButton) => FloatingToolbarButton.isOn(floatingToolbarButton)).getOr(false),
```

Comment:
```suggestion
        memFloatingToolbarButton.getOpt(toolbar).map(FloatingToolbarButton.isOn).getOr(false),
```
Could we get away with not wrapping the function?